raisinbread/README.md

```diff
@@ -64,6 +64,12 @@ correct `host`, `url` and `base` values
 > The password of the `admin` user on the RB database is `demo20!7`
 
 
+#### Getting the imaging files
+MCIN members can get access to the imaging files by putting in a request on redmine
```

Comment:
given what I read on redmine, people should automatically get access without putting in a request no ?

https://redmine.cbrain.mcgill.ca/issues/14481

Reply:
ooops. I updated the doc before Pam updated all the VM on Friday and forgot to update the doc accordingly. 

The README.md is updated in the next commit. Thanks!